async/index.browser.js

```diff
@@ -7,36 +7,30 @@ var crypto = self.crypto || self.msCrypto
 // Despite the fact the source code is quite long, its entropy
 // is low and there are lots of duplicates - just what compressors
 // like GZIP and Brotli likes the best.
-var i
-var url = '_-' + String.fromCharCode(
-  // ASCII codes for 0...9
-  i = 48, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1,
-  i += 1, i += 1,
+var url = '-_'
 
-  // ASCII codes for A...Z
-  i += 8, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1,
-  i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1,
-  i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1, i += 1,
-  i += 1, i += 1,
+var i = 36
+while (i--) {
+  url += i.toString(36)
```

Comment:
Let’s comment `36` argument of `toString`

Reply:
Done